test/FilterTest.php

```diff
@@ -74,4 +74,16 @@ public function testExecutesAllTasksOnException() {
             $this->assertStringEqualsFile($files[2][1], '2');
         }
     }
+
+    public function testFilterWithInvalidFlag() {
```

Comment:
This doesn't test an invalid flag, it rather tests a `null` callback. Could you move that to a separate PR?

Reply:
All right, I set the error testing method.
How about the ```testFilterWithNullCallable``` method.

Reply:
Sounds fine. :+1: